app/uk/gov/hmrc/trackingconsentfrontend/controllers/CookieSettingsController.scala

```diff
@@ -17,22 +17,20 @@
 package uk.gov.hmrc.trackingconsentfrontend.controllers
 
 import javax.inject.{Inject, Singleton}
-
 import play.api.mvc._
 import uk.gov.hmrc.play.bootstrap.controller.FrontendController
 import uk.gov.hmrc.trackingconsentfrontend.config.AppConfig
-import uk.gov.hmrc.trackingconsentfrontend.views.html.hello_world
+import uk.gov.hmrc.trackingconsentfrontend.views.html.CookieSettings
 
 import scala.concurrent.Future
 
 @Singleton
-class HelloWorldController @Inject()(appConfig: AppConfig, mcc: MessagesControllerComponents)
+class CookieSettingsController @Inject()(appConfig: AppConfig, mcc: MessagesControllerComponents, cookieSettings: CookieSettings)
     extends FrontendController(mcc) {
 
   implicit val config: AppConfig = appConfig
 
-  val helloWorld: Action[AnyContent] = Action.async { implicit request =>
-    Future.successful(Ok(hello_world()))
+  val cookie_settings: Action[AnyContent] = Action.async { implicit request =>
```

Comment:
camelCase is the default for Scala. Please don't use underscores, as it makes me think there must be something special going on! :)
If you're concerned about name clash, I'd recommend changing the CookieSettings class to be called CookieSettingsPage - that's the pattern we've used in the Catalogue too.

Reply:
Excellent, I much prefer camel case :-)